script/hassfest/model.py

```diff
@@ -74,6 +74,11 @@ def domain(self) -> str:
         """Integration domain."""
         return self.path.name
 
+    @property
+    def core(self) -> bool:
+        """Core integration."""
+        return "homeassistant/components" in self.path.as_posix()
```

Comment:
`in` or `starswith`?

Reply:
starswith are probably more correct :+1: